websites/N/Neko-sama/metadata.json

```diff
@@ -4,7 +4,7 @@
 		"name": "Mokocchi",
 		"id": "503568040001273877"
 	},
-	"service": "Neko-sama.fr",
+	"service": "Neko-sama",
```

Comment:
if you change this you must change the folder name too, and upload imgur links again for the logo and thumbnail

Reply:
I'm pretty sure I changed Folder name from "websites/N/Neko-sama.fr/" to "websites/N/Neko-sama/" or that's not what you meant.

For imgur links, why did I have to repost them ? Does this mean that create another presence ?

Thanks for your review :)

Reply:
Basically when you edit a presence service name, you delete the old presence, and make a new presence with the new name.
So any users that may have installed the presence will have to reinstall too.

Reply:
Okay so I'll keep Neko-sama.fr